tests/phpunit/Integration/Maintenance/RunImportTest.php

```diff
@@ -38,7 +38,7 @@ protected function tearDown(): void {
 
 	public function testRun() {
 		$maintenanceRunner = $this->runnerFactory->newMaintenanceRunner(
-			'SMW\Maintenance\RunImport'
+			'\SMW\Maintenance\runImport'
```

Comment:
Needs to update the class in `src\MediaWiki\Hooks.php` as well

Reply:
Thanks, done.